frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -247,29 +298,137 @@ const useNavigationMode = () => {
+                >
                     <StyledProjectIcon />
                 </DynamicListItem>
-                <DynamicListItem href='/search' text='Search'>
+                <DynamicListItem href='/search' text='Search' onClick={onClick}>
                     <SearchIcon />
                 </DynamicListItem>
-                <DynamicListItem href='/playground' text='Playground'>
+                <DynamicListItem
+                    href='/playground'
+                    text='Playground'
+                    onClick={onClick}
+                >
                     <PlaygroundIcon />
                 </DynamicListItem>
-                <DynamicListItem href='/insights' text='Insights'>
+                <DynamicListItem
+                    href='/insights'
+                    text='Insights'
+                    onClick={onClick}
+                >
                     <InsightsIcon />
                 </DynamicListItem>
             </List>
+        );
+    };
+
+const ConfigureNavigationList: FC<{
+    routes: INavigationMenuItem[];
+    mode: 'mini' | 'full';
+    onClick?: () => void;
+}> = ({ routes, mode, onClick }) => {
+    const DynamicListItem = mode === 'mini' ? MiniListItem : FullListItem;
+
+    return (
+        <List>
+            {routes.map((route) => (
+                <DynamicListItem
+                    href={route.path}
+                    text={route.title}
+                    onClick={onClick}
+                >
+                    <IconRenderer path={route.path} />
+                </DynamicListItem>
+            ))}
+        </List>
+    );
+};
+
+const AdminNavigationList: FC<{
+    routes: INavigationMenuItem[];
+    mode: 'mini' | 'full';
+    badge?: ReactNode;
+    onClick?: () => void;
+}> = ({ routes, mode, onClick, badge }) => {
+    const showBadge = useShowBadge();
+    const DynamicListItem = mode === 'mini' ? MiniListItem : FullListItem;
+
+    return (
+        <List>
+            {routes.map((route) => (
+                <DynamicListItem
+                    onClick={onClick}
+                    href={route.path}
+                    text={route.title}
+                    badge={
+                        showBadge(route?.menu?.mode) ? (
+                            <EnterprisePlanBadge />
+                        ) : null
+                    }
+                >
+                    <IconRenderer path={route.path} />
+                </DynamicListItem>
+            ))}
+        </List>
+    );
+};
+
+const OtherLinksList = () => {
+    const { uiConfig } = useUiConfig();
+
+    return (
+        <List>
+            {uiConfig.links.map((link) => (
+                <ExternalFullListItem href={link.href} text={link.value}>
+                    <IconRenderer path={link.value} />
+                </ExternalFullListItem>
+            ))}
+            <SignOutItem />
+        </List>
+    );
+};
+
+export const MobileNavigationSidebar: FC<{ onClick: () => void }> = ({
```

Comment:
Mobile sidebar is composed without accordion and without parent Box because we put it inside a drawer that is already a type of Box